lib/connect.js

```diff
@@ -1,3 +1,5 @@
+/** @import HyperDHT from '../index' */
+/** @import UDXStream from 'udx-native/lib/stream' */
```

Comment:
I find these annotations immensely helpful, because it enables language-server based IDEs to link everything. I'm happy to remove them from this PR, or to fully annotate the holepunch ecosystem so IDEs grok it.

Reply:
lets keep that separate, already complex stuff here